src/PDO/Builder.php

```diff
@@ -87,4 +91,21 @@ protected function _hasOptions(): bool
     {
         return $this->_exists(self::PROP_OPTIONS);
     }
-}
\ No newline at end of file
+
+    public function setPort(int $port): BuilderInterface
+    {
+        $this->_create(self::PROP_PORT, $port);
+
+        return $this;
+    }
+
+    protected function _getPort(): int
+    {
+        return $this->_read(self::PROP_PORT);
+    }
+
+    protected function _hasPort(): bool
```

Comment:
Is this even necessary since we have a default of 0?

Reply:
maybe it that default can be overridden somehow?